src/main/java/com/hubspot/jinjava/el/ext/JinjavaBeanELResolver.java

```diff
@@ -20,7 +20,7 @@
   private static final Set<String> RESTRICTED_METHODS = ImmutableSet.<String> builder()
       .add("clone")
       .add("hashCode")
-      .add("getClass")
```

Comment:
let's leave this in, layers are good

Reply:
ok, I took it out mostly so I could test the new method of blocking, but I'll put it back.